### ISSUE TITLE
Install IIIF and OpenSeadragon...

### ISSUE THREAD

**Issue by jamesvanmil** [jamesvanmil/house-of-refuge#19]
...and figure out how to provide direct access to Admission results from the original images, on the show page.

**Comment by jamesvanmil** [comment]
Depends on having images to work with (#12)

**Comment by crowesn** [comment]
Are we still interested in this?

**Comment by jamesvanmil** [comment]
Yes, but not in the short-term - running a IIIF server will probably take some additional server resources (or partnership with UCL)
